core/lib/utils/option_param_mixin.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:dartz/dartz.dart';
+
+mixin OptionParamMixin {
+  T? getOptionParam<T>(Option<T?>? option, T? defaultValue) {
+    if (option != null) {
+      return option.toNullable();
+    } else {
+      return defaultValue;
+    }
+  }
+}
```

Comment:
If possible, use it for `search_filter`. Note: We have created it in the search filters, please save it to one place for future use.

Reply:
done